source/line regulation/configure_dc_power.py

```diff
@@ -0,0 +1,342 @@
+from enum import Enum
+
+import hightime
+from ni_measurementlink_service._internal.stubs.ni.protobuf.types.xydata_pb2 import DoubleXYData
+from nidcpower import (
+    Session, Sense, SourceMode, OutputFunction, Error, Event, MeasureWhen, MeasurementTypes, TriggerType
+)
+
+
+# Mode of operation ENUM
+class ModeOfOperation(Enum):
+    Power_On_DUT = 0
+    PerformMeasurement = 1
+    Power_Off_DUT = 2
+    pass
+
+
+# Sweep type indicator ENUM
+class SweepType(Enum):
+    Linear = 0
+    Logarithmic = 1
+    pass
+
+
+# function to take voltage and current levels and return status string
+def format_power_on_result(voltage_level: float, current_level: float) -> str:
```

Comment:
Can you limit both Voltage Level and Current Level to 3 decimal points

Reply:
Changed them to display only 3 digits after decimal point.